browser/browser/profiles.ftl

```diff
@@ -114,22 +116,102 @@ book-avatar-alt =
     .alt = Book
 briefcase-avatar-alt =
     .alt = Briefcase
+# Canvas refers to an artist's painting canvas, not the general material
+canvas-avatar-alt =
+    .alt = Canvas
+# Craft refers to hobby arts and crafts, represented by a button/fastener commonly found on clothing like shirts
+craft-avatar-alt =
+    .alt = Craft
+custom-avatar-alt = Custom
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1977144

CC @eemeli 

Automation here is where the bug might actually be. There is now a version without attribute here, and one with attribute on line 174.

Reply:
@eemeli - I'm going to try to test this locally tomorrow to see if we get expected results with the latest version (v.0.8.1)